applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -38,7 +38,7 @@ namespace Kratos
 
     enum class SofteningType {Linear = 0, Exponential = 1, HardeningDamage = 2};
 
-    enum class TangentOperatorEstimation {Analytic = 0, FirstOrderPerturbation = 1, SecondOrderPerturbation = 2, Secant = 3};
+    enum class TangentOperatorEstimation {Analytic = 0, FirstOrderPerturbation = 1, SecondOrderPerturbation = 2, Secant = 3, SecondOrderPerturbationV2 = 4};
```

Comment:
This is no longer used in structural right?
Can you move it to claw?

Reply:
I've moved ALL the variables only used in the CL :)